AbletonMCP_Remote_Script/handlers/automation.py

```
@@ -0,0 +1,350 @@
+        if clip_len <= 0:
+            return {"has_automation": False, "parameter": parameter_name, "reason": "Clip has zero length"}
+
+        points = []
+        step = clip_len / num_samples
+        for i in range(num_samples + 1):
+            t = i * step
+            try:
+                val = envelope.value_at_time(t)
+                points.append({"time": round(t, 4), "value": round(val, 4)})
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)